plugins/github/app/models/github_deployment.rb

```diff
@@ -26,12 +26,22 @@ def deployment_options
     {
       accept: DEPLOYMENTS_PREVIEW_MEDIA_TYPE,
       force: true,
-      payload: {deployer: @deploy.user.name}.to_json,
+      payload: payload.to_json,
       environment: @stage.name,
       description: @deploy.summary
     }
   end
 
+  def payload
+    {
+      deployer: @deploy.user.name,
+      deployer_email: @deploy.user.email,
+      buddy: @deploy.buddy_name,
+      buddy_email: @deploy.buddy_email,
+      production: @stage.production?
```

Comment:
@grosser I could. I'm just interested in `production?` in our use case. How would the environment look like?